### ISSUE TITLE
Camera should turn when roller turns

### ISSUE THREAD

**Issue by Lipsch** [Lipsch/roller#6]
Check out if rotating the camera around the roller depending on roller's current direction. 

- What should happen when the roller changes the direction by 180°? (stop and roll back)

**Comment by Der-Die-Das** [comment]
I'd say the camera should always look to the rollers velocity direction.
Maybe add some improvements relating to smoothness.

**Comment by Lipsch** [comment]
I agree. The question is what should happen then the roller stops and rolls backward. Should the camera rotate around or switch over on top of the roller.

**Comment by Der-Die-Das** [comment]
I think rotating aorund the ball would be less confusing, since the camera would have to rotate through two axis.

**Comment by Lipsch** [comment]
Agree. Let's try it that way

**Comment by Lipsch** [comment]
Created #15 as a follow-up.
